backend/main.go

```diff
@@ -185,3 +196,47 @@ func (w *Worker) processStellarWithdrawalRequest(sr store.SignatureRequest) erro
 
 	return nil
 }
+
+func (w *Worker) processEthereumRefundRequest(sr store.SignatureRequest) error {
+	if sr.DepositChain != store.Ethereum {
+		return fmt.Errorf("deposits from %v are not supported", sr.DepositChain)
+	}
+	deposit, err := w.Store.GetEthereumDeposit(context.TODO(), sr.DepositID)
+	if err != nil {
+		return errors.Wrap(err, "error getting ethereum deposit")
+	}
+
+	if err = w.EthereumRefundValidator.CanRefund(context.TODO(), deposit); err != nil {
+		return errors.Wrap(err, "error validating refund conditions")
+	}
+
+	amount, ok := new(big.Int).SetString(deposit.Amount, 10)
+	if !ok {
+		return errors.Errorf("cannot convert value in wei to bit.Rat: %s", deposit.Amount)
+	}
+
+	expiration := int64(math.MaxInt64)
```

Comment:
Is it ever going to change? If not, maybe we don't need to store it in a DB?

Reply:
the same table will store both withdrawals and refunds. So for the withdrawal case the expiration is 24 hours after the stellar deposit which is not constant